backend/source/config/dashboard.json

```diff
@@ -4,31 +4,27 @@
       "component": "JMP-CHARTS",
       "selected": true,
       "chartList": [
-        [
-          {
-            "type": "chart",
-            "title": "Service level for sanitation in schools",
-            "calc": "jmp",
-            "path": "Sanitation",
-            "span": 12
-          },
-          {
-            "type": "chart",
-            "title": "Service level for Hygiene in schools",
-            "calc": "jmp",
-            "path": "Hygiene",
-            "span": 12
-          }
-        ],
-        [
-          {
-            "type": "chart",
-            "title": "Service level for drinking water in schools",
-            "calc": "jmp",
-            "path": "Water",
-            "span": 24
-          }
-        ]
+        {
```

Comment:
Hi @navins94 ,
Instead of reordering like this, I think it better we add order param to config, then we can order the chart on FE by that.